src/main/java/com/rockset/client/api/CustomDocumentsApi.java

```diff
@@ -0,0 +1,92 @@
+package com.rockset.client.api;
+
+import com.rockset.client.ApiClient;
+import com.rockset.client.model.AddDocumentsRequest;
+import com.rockset.client.model.AddDocumentsResponse;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class CustomDocumentsApi extends DocumentsApi {
```

Comment:
Should this be a package scoped class? I assume library consumers shouldn't interact with this class directly?

Also, maybe add a comment that this method extends the parent DocumentsApi class that gets autogenerated.